packages/xo-web/src/common/xo/index.js

```diff
@@ -2919,3 +2919,17 @@ export const unlockXosan = (licenseId, srId) =>
 // Support --------------------------------------------------------------------
 
 export const checkXoa = () => _call('xoa.check')
+
+export const closeTunnel = () =>
+  _call('xoa.closeSupportTunnel')::tap(subscribeTunnelState.forceRefresh)
+
+export const openTunnel = () =>
+  _call('xoa.openSupportTunnel')::tap(() => {
+    subscribeTunnelState.forceRefresh()
+    // timeout: to bring the tunnel state as fast as possible.
```

Comment:
"After 1s, we most likely got the tunnel ID and we don't want to wait another 5s to show it to the user"